soco/core.py

```diff
@@ -1307,6 +1308,25 @@ def get_current_transport_info(self):
 
         return playstate
 
+    def get_available_actions(self):
```

Comment:
This look great. I have just two quick questions related to the function definition.

1) Should this be a property. Python tends to favor them for trivial information items and I think this might be one of those.
2) Does this work on a slave or should it have the `only_on_master` decorator?